<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AngKAddeMMCHuaweiFile</class>
 <widget class="QWidget" name="AngKAddeMMCHuaweiFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AngKAddeMMCHuaweiFile</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="acxmlFile">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>ACXML File:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="acxmlEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="acxmlButton">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="binFile">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>ACIMG File:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="binEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="binButton">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>238</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>238</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
